### ISSUE TITLE
[Accessibility Audit] Frontend Platform: Commits page

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#34423]
### Steps to replicate journey

Taken from the [user journey list document](https://docs.google.com/document/d/1rSp77P0UDY6ewHq6iXmW0nuilBtkMW05n1ciMB8gQ3w/edit?usp=sharing). Use this for further context.

Note: This page doesn’t have significant changes after the repository redesign. Double check this is still the case before actioning these user journeys in any way.

View commits:

- Go to commits tab
- Click on a commit message to navigate to that commit diff
  - Click on the “...” icon to expand long commit messages
- Click on a commit hash to navigate to that commit diff
- Click on the copy icon to copy the fully commit SHA hash to your clipboard
- Click on the files icon to navigate back to the repository tree page with the selected commit used as the repository revision.
- Click “Show more” to load more commits

### How to audit

Follow the instructions here: [Auditing a user journey](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#auditing-a-user-journey)

**Note:** We are planning on arranging thorough third-party audit in the future, so our focus here is just to find the _majority_ of accessibility issues. Don't worry if you feel you haven't found 100% of potential issues, it is better to focus on the core essentials to complete the journey rather than spending lots of time going through every possible problem.

### How to raise discovered problems

Follow the instructions here: [Raising an accessibility bug](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#raising-a-bug)

### Any other questions?

See the [audit FAQ](https://docs.google.com/document/d/1nCi7TvYRpj9eG0_QtgqK1c4L28EiJKVuxyuvg1lAGgo/edit#) - add your question if it isn't there already!

**Comment by umpox** [comment]
Audit complete, issues are linked
